### PR TITLE
Added Inductiva wiki files to docs

### PR DESCRIPTION
**Pull request by maiyahershey**
Pasted all .md files of inductiva's wiki to prepare them for edits

### CODE REVIEW

File: docs/API-access-tokens.md
Comment (sarmento) [inline docs/API-access-tokens.md:12]: I would like to deprecate this method on this release. It bad practice to have keys in the code.

@sergiomsantos @IvanPombo can we remove all examples where we do this?

As a plus, our exemples can be run automatically because there will be no need to substitute the key in the code snippet.

File: docs/Install.md
Comment (sarmento) [inline docs/Install.md:3]: Maybe we want to have the install info on the top page, and the trouble shooting info on this separate page.

File: docs/Install.md
Comment (sarmento) [inline docs/Install.md:20]: This is not true anymore, is it?
We don't have these extras anymore, right?
@IvanPombo can you confirm?

File: docs/Machines.md
Comment (sarmento) [inline docs/Machines.md:7]: Are we going from 1 "#" (Heading 1) to  3 "#" (Heading 3) for any specific reason.

File: docs/Machines.md
Comment (sarmento) [inline docs/Machines.md:38]: This does not exist anymore in this base package.
We will need to change this example asap.

File: docs/Running-multiple-simulations-in-parallel.md
Comment (sarmento) [inline docs/Running-multiple-simulations-in-parallel.md:16]: This does not exist anymore in this package either

File: docs/Scenarios.md
Comment (sarmento) [inline docs/Scenarios.md:1]: I think we don't have these here, at least in this version.

File: docs/Simulators.md
Comment (sarmento) [inline docs/Simulators.md:29]: A couple of things here:

1)  inductiva.utils.files.download_from_url is too deep. It should be just  inductiva.utils.download_from_url or  inductiva.files.download_from_url

2) there is a magic unzip happening here.

File: docs/Simulators.md
Comment (sarmento) [inline docs/Simulators.md:24]: We need to think a bit more about which files are being used as input in these examples.
Ideally, we should, as much as possible, use whatever is already available in the docs/example pages of the simulator, so that users understand what they are running. The example files below seem opaque.

File: docs/Start-simulating.md
Comment (sarmento) [inline docs/Start-simulating.md:5]: No more wind tunnel in this Repo.

File: docs/_Sidebar.md
Comment (sarmento) [inline docs/_Sidebar.md:20]: In this release, we have no scenarios. In fact, the base package will probably not point to any scenario from this side bar.

File: docs/scenarios/Build-your-own-scenario.md
Comment (sarmento) [inline docs/scenarios/Build-your-own-scenario.md:1]: I doubt we need this here for now.

File: docs/scenarios/Build-your-own-scenario.md
Comment (sarmento) [inline docs/scenarios/Build-your-own-scenario.md:9]: This is fine. I don't think we need to introduce scenarios for now. Let's just talk about templates. Or, let's actually have a section on "generalizing your simulation" or "running variations of your simulation". I think if we can avoid the word scenario for now, it's great.

File: docs/scenarios/Build-your-own-scenario.md
Comment (sarmento) [inline docs/scenarios/Build-your-own-scenario.md:28]: We really don't need this here for this release.

File: docs/scenarios/Coastal-Area.md
Comment (sarmento) [inline docs/scenarios/Coastal-Area.md:1]: No scenarios here. We can remove this one.

File: docs/scenarios/Fluid-Tank.md
Comment (sarmento) [inline docs/scenarios/Fluid-Tank.md:1]: ditto...

File: docs/scenarios/Protein-Solvation.md
Comment (sarmento) [inline docs/scenarios/Protein-Solvation.md:1]: the same. no need for this here.

File: docs/scenarios/Wind-Tunnel.md
Comment (sarmento) [inline docs/scenarios/Wind-Tunnel.md:1]: pls remove.

**Review by sarmento — COMMENTED** [review]
Great first push.
I am sending just a first pass, mostly for making sure we select the right material to add to the docs. I would do a fine grained pass later, but at this stage I think the most important thing is to completely remove anything that has to do with scenarios. We don't need any of those concepts for now, and in fact, non of that code lives in this repo anymore (or is about to get removed). I suggest we try just removing all that info now before we can proceed to a finer level of detail type of revision.
Thanks!!

File: docs/Install.md
Comment (IvanPombo) [reply docs/Install.md:20]: Yes, this can be removed confidently!

File: docs/Simulators.md
Comment (IvanPombo) [reply docs/Simulators.md:24]: These files are almost all taken from examples available online. However, I agree with the statement of explaining the simulation being run, even though we don't show any immediate results.

File: docs/simulators/XBeach.md
Comment (IvanPombo) [inline docs/simulators/XBeach.md:24]: `task.download_outputs()`

File: docs/simulators/Reef3D.md
Comment (IvanPombo) [inline docs/simulators/Reef3D.md:22]: The arg is now named as `on`.

**Review by IvanPombo — COMMENTED** [review]
Wonderful @maiyahershey! Left a few extra comments to complement Sarmento.

**Review by sergiomsantos — COMMENTED** [review]
@maiyahershey
Great first contribution.
There are a lot of things that need to be revised as we go along, but this is a huge step towards cleaning the docs and bringing them up-to-date.
To move forward, this is what I suggest:
1 - change the base branch from `main` to `development`;
2 - I'll approve the PR once we're able to merge this PR to `development`;
3 - Out of `development`, create a new branch to address the comments from this PR. Let's assume that the sole purpose of this PR is to mirror the contents of the wiki. All changes to it will come from future PRs

**Review by sergiomsantos — APPROVED** [review]
Let's proceed with the plan we discussed previously:
1. merge this PR to `development`;
2. open smaller PRs to address the comments on this PR and start iterating over the actual doc content.

**Comment by maiyahershey** [comment]
@sergiomsantos Amazing - I merged this with development and will proceed with the smaller PRs to address the comments and improve the repo.

**Pull request merged**
